js/customscripts.js

```diff
@@ -22,16 +22,39 @@ $( document ).ready(function() {
 
     initSidebarToggle();
 
+    initSidebarAccordion();
+
+    initFeedbackForm();
+
+    initCopyText();
+});
+
+function initCopyText() {
+    jQuery('code').on('click', function(e){
+    let copyText = this,
```

Comment:
```suggestion
        let copyText = this,
```

Reply:
let's skip it. script was totally changed in next branch